zimbra/src/main/java/fr/openent/zimbra/cron/RecallMailCron.java

```diff
@@ -0,0 +1,36 @@
+package fr.openent.zimbra.cron;
+
+import fr.openent.zimbra.service.RecallMailService;
+import fr.openent.zimbra.worker.RecallMailWorker;
+import io.vertx.core.Handler;
+import io.vertx.core.eventbus.EventBus;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+public class RecallMailCron implements Handler<Long> {
+    private static final Logger log = LoggerFactory.getLogger(RecallMailCron.class);
+
+    private final RecallMailService recallMailService;
+    private final EventBus eb;
+
+    public RecallMailCron(RecallMailService recallMailService, EventBus eb) {
+        this.recallMailService = recallMailService;
+        this.eb = eb;
+    }
+
+    @Override
+    public void handle(Long event) {
+        log.info("[ZimbraConnector@RecallCron] RecallCron started");
+        final JsonObject message = new JsonObject();
+        message.put("action", RecallMailWorker.ACTION_REFILL_QUEUE);
+        eb.request(RecallMailWorker.RECALL_MAIL_HANDLER_ADDRESS, message, handler -> {
```

Comment:
Tu peux peut-être mettre en place un handler pour les messages response (cf exemple : https://github.com/OPEN-ENT-NG/nextcloud/blob/dev/src/main/java/fr/openent/nextcloud/helper/MessageResponseHandler.java#L11)

(à adapter avec asyncResult - pas besoin d'utiliser either)
Comme ça tu pourrais l'appeler comme ça : 

```suggestion
        eb.request(RecallMailWorker.RECALL_MAIL_HANDLER_ADDRESS, message, MessageResponseHandler.messageJsonObjectAsyncHandler(event -> {
             // content
        }));
```